code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -417,13 +419,15 @@
 	interface_desc = "Leave your mark."
 	engage_string = "Toggle stamp type"
 	usable = 1
-	var/iastamp
-	var/deniedstamp
+	var/obj/iastamp			//Theese were just vars, but any device would need to be an object
+	var/obj/deniedstamp //Stops assigning non-objects to theese vars, which probably would break quite a bit.
```

Comment:
As the utility modules assign the flag ``ABSTRACT`` to the ``obj`` they use, and i don't think that simple variables have flags, i did change the type so that on creation, i can also change the flag for the inactive stamp, else we would have one stamp that doesn't get placed if you activate the module onto a table, but the other does.